test/IISIntegration.FunctionalTests/NtlmAuthentationTest.cs

```diff
@@ -22,29 +21,23 @@ public NtlmAuthenticationTests(ITestOutputHelper output) : base(output)
         {
         }
 
-        [ConditionalTheory]
-        [OSSkipCondition(OperatingSystems.Linux | OperatingSystems.MacOSX)]
-        [FrameworkSkipCondition(RuntimeFrameworks.CoreCLR)]
-        [InlineData(RuntimeFlavor.Clr, RuntimeArchitecture.x64, ApplicationType.Portable)]
-        public Task NtlmAuthentication(RuntimeFlavor runtimeFlavor, RuntimeArchitecture architecture, ApplicationType applicationType)
```

Comment:
None of the NTLM tests are set up to run today. When enabled, they fail on my box:
```
Error Message:
 Assert.Equal() Failure
Expected: Unauthorized
Actual:   OK
Stack Trace:
   at Microsoft.AspNetCore.Server.IISIntegration.FunctionalTests.NtlmAuthenticationTests.<NtlmAuthentication>d__3.MoveNext() in D:\work\IISIntegration\test\IISIntegration.FunctionalTests\NtlmAuthentationTest.cs:line 123
```